Added how to install next-auth (#5040)
nextauthjs/next-auth: docs/docs/getting-started/example.md

## Existing Project

### Install NextAuth

```

> npm install --save next-auth

Reviewer: ```suggestion
npm i next-auth
```

```

or

```
yarn add next-auth
```

> :::TIP
> If you are using TypeScript in your project, you don't need to types definition for `next-auth`. NextAuth has its own types definitions which has been done for you! To learn more about TypeScript for `next-auth`, check out the [TypeScript documentation](/getting-started/typescript)
> :::

Reviewer: ```suggestion
:::info
If you are using TypeScript, NextAuth comes with its types definitions within the package. To learn more about TypeScript for `next-auth`, check out the [TypeScript documentation](/getting-started/typescript)
:::
```
